src/main/java/com/google/sps/database/DatabaseImpl.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Implementation for DatabaseImpl class
+ *  Current public functions at a glance:
+ *  test()
+ *  addVideo()
+ *  addClause()
+ *  addClauses()
+ *  addMetadata()
+ *  getAllKeywords()
+ *  getTimesForKeyword()
+ * 
+ */
+
+package com.google.sps.database;
+
+import com.google.appengine.api.datastore.*;
+import java.util.*;
+
+public class DatabaseImpl implements DatabaseInterface {
```

Comment:
Just a note for other reviewers, these files are outdated. Database updated functionality is in the `database` branch.